### ISSUE TITLE
Add comment feature

### ISSUE THREAD

**Issue by willbed34** [chris-koch-penn/polylang.io#2]
Suggesting a comment feature where a user can comment a single or multiple lines of code using ctrl-/

**Comment by chris-koch-penn** [comment]
Agreed, I'll leave this issue open until I get the time to implement that feature.
